sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -281,9 +281,11 @@ def test_univariate_pls_regression():
 
     clf = pls_.PLSRegression()
     # Compare 1d to column vector
-    model1 = clf.fit(X, Y[:, 0]).coef_
-    model2 = clf.fit(X, Y[:, :1]).coef_
-    assert_array_almost_equal(model1, model2)
+    # issue #8856
+    ndim_predict_1d = clf.fit(X, Y[:, 0]).predict(X).ndim
+    ndim_predict_2d = clf.fit(X, Y[:, :1]).predict(X).ndim
+    assert_equal(ndim_predict_1d, 1)
```

Comment:
You are weakening the test by only checking the dimension and not the content anymore. I think you should still check that the content is fine on top of your dimension check, one is 2d array but with a single column with the same content as the first one.

Maybe you want to check the `coef_` (as in the old test) as well as the `predict` (as in your current test).

Reply:
You are right. I corrected this